stripe/_account.py

```diff
@@ -1,17 +1,16 @@
 # -*- coding: utf-8 -*-
```

Comment:
So I know we talked about potentially putting all these resources into an `_api` subpackage (potentially cleaner / easier to navigate). Just to confirm, the reason not to have an `_api` subpackage is because we need namespace packages like `tax` to be on the top-level , so that `from stripe.tax import ...` will work?

Reply:
Correct.  But I also quite like how exports and the directory structure align it's easier to see type conflicts and in general think about public API this way.